packages/README.md

```diff
@@ -0,0 +1,107 @@
+#Liota Package Manager
```

Comment:
There is already a Readme under liotad folder. https://github.com/vmware/liota/blob/fe00dc066feeb3eb9508bb02780aad5b4e0a8ad8/packages/liotad/README.md

If you want to move it here, we better delete the old one. It will be difficult to manage consistencies of multiple Readme files.

Reply:
done